test/integration/install/test_install.py

```diff
@@ -77,6 +77,77 @@ def test_install_tt_specific_version(tt_cmd, tmpdir):
     assert re.search(r"Tarantool CLI version 1.0.0", start_output)
 
 
+@pytest.mark.slow
+def test_install_tt_specific_commit(tt_cmd, tmpdir):
+    configPath = os.path.join(tmpdir, config_name)
+    # Create test config
+    with open(configPath, 'w') as f:
+        f.write('tt:\n  app:\n    bin_dir:\n    inc_dir:\n')
+
+    # Install latest tt.
+    install_cmd = [tt_cmd, "--cfg", configPath, "install", "tt", "2df3077"]
+    instance_process = subprocess.Popen(
+        install_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+
+    # Check that the process shutdowned correctly.
+    instance_process_rc = instance_process.wait()
+    assert instance_process_rc == 0
+
+    installed_cmd = [tmpdir + "/bin/tt", "version"]
+    installed_program_process = subprocess.Popen(
+        installed_cmd,
+        cwd=tmpdir + "/bin",
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    start_output = installed_program_process.stdout.readline()
+    assert re.search(r"Tarantool CLI version 1.0.1", start_output)
+    assert re.search(r"commit: 2df3077", start_output)
+
+
+@pytest.mark.slow
+def test_install_tarantool_commit(tt_cmd, tmpdir):
+    config_path = os.path.join(tmpdir, config_name)
+    # Create test config.
+    with open(config_path, "w") as f:
+        yaml.dump({"tt": {"app": {"bin_dir": "", "inc_dir": "./my_inc"}}}, f)
+
+    tmpdir_without_config = tempfile.mkdtemp()
+
+    # Install latest tarantool.
+    install_cmd = [tt_cmd, "--cfg", config_path, "install", "-f", "tarantool", "00a9e599"]
+    instance_process = subprocess.Popen(
+        install_cmd,
+        cwd=tmpdir_without_config,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+
+    # Check that the process was shutdowned correctly.
+    instance_process_rc = instance_process.wait()
+    assert instance_process_rc == 0
+    installed_cmd = [tmpdir + "/bin/tarantool", "-v"]
+    installed_program_process = subprocess.Popen(
+        installed_cmd,
+        cwd=os.path.join(tmpdir, "/bin"),
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+
+    run_output = installed_program_process.stdout.readline()
+    assert re.search(r"Tarantool", run_output)
+    assert os.path.exists(os.path.join(tmpdir, "my_inc", "include", "tarantool"))
```

Comment:
Please, add check for file and dirs existing: tarantool_<hash> in bin and include dirs.

Reply:
fixed